packages/web/src/app/admin/products/ProductForm.component.tsx

```diff
@@ -0,0 +1,337 @@
+                        />
+                    </div>
+                    <div>
+                        <label htmlFor="weight" className="block text-sm font-medium leading-6 text-gray-900">
+                            Gewicht (in kg)
+                        </label>
+                        <input
+                            type="number"
+                            step="0.1"
+                            name="weight"
+                            id="weight"
+                            value={formData.weight}
+                            onChange={handleChange}
+                            required
+                            className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                        />
+                    </div>
+                </div>
+
+                {/* Color and Size Selects */}
+                <div className="grid grid-cols-1 gap-6 sm:grid-cols-2">
+                    <div>
+                        <label htmlFor="color" className="block text-sm font-medium leading-6 text-gray-900">
+                            Farbe
+                        </label>
+                        <select
+                            id="color"
+                            name="color"
+                            value={formData.color}
+                            onChange={handleChange}
+                            className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                            <option>yellow</option>
+                            <option>red</option>
+                            <option>green</option>
+                            <option>blue</option>
+                        </select>
+                    </div>
+                    <div>
+                        <label htmlFor="size" className="block text-sm font-medium leading-6 text-gray-900">
+                            Größe
+                        </label>
+                        <select
+                            id="size"
+                            name="size"
+                            value={formData.size}
+                            onChange={handleChange}
+                            className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                            <option>s</option>
+                            <option>m</option>
+                            <option>l</option>
+                            <option>xl</option>
+                            <option>xxl</option>
```

Comment:
When using an `enum` you can iterate over the options, instead of writing them down each by each.